crates/matrix-sdk-crypto/CHANGELOG.md

```diff
@@ -17,13 +20,27 @@
   `OlmMachine::backup_machine()::import_backed_up_room_keys()` for backups. The
   `OlmMachine::import_room_keys()` method is now deprecated.
 
-- Add support for secret storage.
```

Comment:
this landed in 372e0d3d98 which predates the changes to `encrypt_room_event_raw` (d9845c1658) and all the other stuff I've moved from the bottom of the file, so moving it down while we're here.